src/Application/RuntimeParameters.php

```diff
@@ -0,0 +1,46 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Psh\Application;
+
+class RuntimeParameters
+{
+    /**
+     * @var array
+     */
+    private $appParams;
+
+    /**
+     * @var array
+     */
+    private $overwrites;
+
+    /**
+     * @var array
+     */
+    private $commands;
+
+    public function __construct(
```

Comment:
how do you handle arrays? do you normally specify the content? especially for Psalm/PHPStan?

Reply:
usually not, because there are a lot of tests covering the cases possible throughout the application